### ISSUE TITLE
Get Tables sync action support

### ISSUE THREAD

**Issue by Halama** [keboola/ex-teradata#10]

**Comment by Halama** [comment]
Nevím jestli je ten formát někde popsaný jak má vypadat. Nejvíce bude vědět asi @pivnicek

**Comment by pivnicek** [comment]
Jo, existing keys are here https://sites.google.com/keboola.com/devel-internal/storage/metadata.

The `getTables` returns these (or a subset of these) without the `KBC.` prefix. 

The api response structure as you have in the issue description is the same for all extractors.

**Comment by Actimel** [comment]
Prozatim jsou pridane zakladni data o tabulce/sloupci, zbytek metadat musim v teradate jeste dopatrat. Nastepavani tabulek a sloupcu uz funguje.
